autobahn/util.py

```diff
@@ -463,7 +463,10 @@ def generate_serial_number():
     # first call to this function, as a floating point number, based on the
     # Win32 function QueryPerformanceCounter(). The resolution is typically
     # better than one microsecond
-    _rtime = time.clock
+    if sys.version_info >= (3, 8):
```

Comment:
`perf_counter` was added in Python 3.3. Why we can't use it for all Python 3.3+ line? 
`time.clock` produces a warning on `<Python 3.8`